### DISCUSSION TITLE
`defineField` does not allow `__experimental_search`

### DISCUSSION THREAD

**Discussion by alexeygolev** [sanity-io/sanity#3737] [Sanity Studio Support]
Is `__experimental_search` still supported? `defineType` does not allow it but it seems to be actually working. So I assume it's something to do with the actual type definition.

**Comment by geball** [comment]
Hi @alexeygolev. We plan to maintain support for `__experimental_search` into the initial launch of `v3`. I believe you're right that this is an issue with the type definitions.
